### DISCUSSION TITLE
rabbitmqctl.bat on Windows Server 2012R2 fails with "no match of right hand value"

### DISCUSSION THREAD

**Discussion by nimeshpatel1605** [rabbitmq/rabbitmq-server#12492] [Other] [answered]
### Describe the bug

I have installed (as admin user) Erlang OTP 26.2.5.3 and RabbitMQ Server 4.0.2 on Windows server 2012R2 and managed to start Rabbit MQ Service too but when tried to start management from "RabbitMQ Command Prompt (sbin dir)" with command "rabbitmq-plugins enable rabbitmq_management" i see error "no match of right hand value". even i tried to run help command also "rabbitmqctl help" also face same error. my local pc all working well.

![image](https://github.com/user-attachments/assets/2a3627d5-0c82-4091-814a-d19baf6f72bc)

![image](https://github.com/user-attachments/assets/b01b4841-c0cc-400d-a430-d380ce807955)

I have also tried re-install RabbitMQ but still same.

Also below are outcome of Set command
![image](https://github.com/user-attachments/assets/d54e068a-c770-40f9-a0ba-a2bfcaf09849)
and log file
[rabbit@WIN-VHN4I1HUNPA.log](https://github.com/user-attachments/files/17286948/rabbit%40WIN-VHN4I1HUNPA.log)

### Reproduction steps

1. Open RabbitMQ Command Prompt (sbin dir)
2. Run rabbitmqctl help
3. Same error "no match of right hand value"
...

### Expected behavior

My local pc
![image](https://github.com/user-attachments/assets/974e62c7-e7a6-48f0-94d6-5060b49247f9)

![image](https://github.com/user-attachments/assets/f98d7471-44e9-4c2f-9780-0d09a36ca35e)

**Comment by mkuratczyk** [comment, marked as answer]
This is a known incompatibility between recent Erlang versions and crazy-old-why-would-you-ever-use-it Windows versions: https://github.com/erlang/otp/issues/8427#issuecomment-2079031524.

Don't use crazy old Windows versions.
